src/app/models/form.server.model.ts

```diff
@@ -395,6 +404,18 @@ MultirespondentFormWorkflowPath.discriminator(
   WorkflowStepDynamicSchema,
 )
 
+MultirespondentFormSchema.post('save', async function (doc, next) {
```

Comment:
Can the admins explicitly remove the respondents from the first steps? If so, ignore this comment.

Reply:
Yes, they can. If an admin wants to change a field/static email from step 1, they would go to the step 1 builder and change it there.  
They can also go to the email notifications setting to unselect step 1 from receiving the email notifications.  

To elaborate, `stepsToNotify` stores the ids of the steps that have been selected (and does not contain what email fields/static emails to send to).
When sending out notification emails, we loop through the steps and extract out the static emails / fields and then forward it to the user.

Reply:
Hm. There will be a funky behaviour here. 

1. Delete Respondent in `Settings > Email notifications > Notify respondents in your workflow`
2. Refresh the page
3. Observe that the field remains empty
4. Add collaborator `ken+1@open.gov.sg` [a]
5. Observe that `Settings > Email notifications > Notify respondents in your workflow` has some value set.

[a] any model method that triggers a `.save()`. Examples of such actions are `updateFormCollaborators`, `updateFormFieldById`, `insertFormField`, `duplicateFormFieldByIdAndIndex`, ...

Reply:
As discussed, will choose the simplest solution: 
- move this to controller -> update `stepsToNotify` without need for txn.